components/service/glean/src/test/java/mozilla/components/service/glean/ping/PingMakerTest.kt

```diff
@@ -139,6 +139,7 @@ class PingMakerTest {
     fun `seq number must be sequential`() {
         // NOTE: Using a "real" ApplicationContext here so that it will have
         // a working SharedPreferences implementation
+        @Suppress("DEPRECATION")
         var applicationContext = RuntimeEnvironment.application.applicationContext
```

Comment:
I think this can become: `var applicationContext = ApplicationProvider.getApplicationContext<Context>()` or simply `var applicationContext = ApplicationProvider.getApplicationContext()` (if it doesn't complain about the casting)